command/agent/command.go

```diff
@@ -278,6 +278,46 @@ func (c *Command) setupAgent(config *Config, logOutput io.Writer) *Agent {
 		return nil
 	}
 
+	if config.ProfilePath != "" {
+		path := config.ProfilePath
+		f, err := os.Open(path)
+		defer f.Close()
+		if err != nil {
+			c.Ui.Error(fmt.Sprintf("Error reading '%s': %s", path, err))
+			return nil
+		}
+		profileconfig, err := DecodeProfile(f)
+		if err != nil {
+			c.Ui.Error(fmt.Sprintf("ProfilePath: %v", err))
```

Comment:
This can probably have the same error as above, `c.Ui.Error(fmt.Sprintf("Error reading '%s': %s", path, err))`